### DISCUSSION TITLE
Conteúdos DB #198

### DISCUSSION THREAD

**Discussion by arturcorreaf** [YuriAlessandro/gerador-ficha-tormenta20#366] [Geral]
### **Novos Poderes**

**Poderes de Bárbaro**

- Brado Amplo
- Brado Assombroso
- Brado Revigorante
- Brado Vitorioso
- Fúria Rodopiante
- Instinto Feral
- Instinto Furioso
- Instinto Indomável
- Instinto Sábio
- Instinto Robusto
- Sem Piedade

**Poderes de Inventor**

- Autômato Alquímico
- Catalisador Composto
- Catalisador Químico
- Compartimento de Carga
- Estilista
- Explicação Científica
- Explorar Fraqueza
- Luthier
- Saraivada
- Traje de Batalha

### **Vilões**
_Npc's desprezíveis para dificultar a vida dos seus jogadores em Tormenta20._

Basicamente são fichas de vilões com suas histórias. Não sei se entraria no Caverna do Saber, mas já deixo aqui também.

- Karmen Roth
- Mesmerizador
- Lady Isoldine, Baronesa Sangrenta
- Rovert Naidr, Taverneiro Traiçoeiro
- Doutora Stella e Ellyn, a Criatura

**Comment by YuriAlessandro** [comment]
Tudo adicionado lá, incluindo os vilões - lá tem uma sessão só para Ameaças, encaixei lá.

Muito obrigado, isso ajuda muito por aqui
